guide/case_study.html

```diff
@@ -1090,10 +1097,10 @@ <h2>Complete Implementation</h2>
     uses substance "R-410A"
       set priorEquipment to 400000 units during year beginning
       initial charge with 1.00 kg / unit for sales
-      retire 1 % each year
+      retire 6 % each year  # roughly equivalent to 15 years
       recharge 10 % each year with 1.00 kg / unit
       set import to 100 units / year during year beginning
-      set manufacture to 40000 units / year during year beginning
+      set manufacture to 40000 units / year during year beginning  # recharge will not reduce when units are specified in a set point - setting manufacture to 100 units means 100 new units will be produced on top of the kg that goes into recharge for prior equipment
```

Comment:
Please break this comment into multiple lines each no longer than 100 characters long.